dashboard/src/components/Header/Menu.scss

```diff
@@ -119,6 +119,7 @@ div .dropdown-menu .dropdown-divider {
   margin-bottom: 0.5rem;
   font-size: smaller;
   text-align: end;
+  white-space: normal;
```

Comment:
Required for the break line after adding the `VMware by Broadcom` text:

![2](https://github.com/vmware-tanzu/kubeapps/assets/11535726/d1109575-52e6-477c-85e3-72ebe1043d4d)